### PR TITLE
[9.0][MIG] account_payment_transfer_reconcile_batch: Migrate to v9

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @tecnativa

### CODE REVIEW

File: account_payment_transfer_reconcile_batch/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Language: es\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. module: account_payment_transfer_reconcile_batch
+#: code:addons/account_payment_transfer_reconcile_batch/models/payment_order.py:46
+#, python-format
+msgid "Nothing to do because the record has been deleted"
+msgstr ""
```
Comment (pedrobaeza) [inline account_payment_transfer_reconcile_batch/i18n/es.po]: Translate this.

**Comment by pedrobaeza** [comment]
Add connector repository to `oca_dependencies.txt` file.

**Comment by sergio-teruel** [comment]
@pedrobaeza There is not oca_dependencies file in this repository, Travis fails because the module connector is not available

File: account_payment_transfer_reconcile_batch/wizard/account_payment_line_create.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+from openerp import api, models
+
+
+class PaymentOrderCreate(models.TransientModel):
+    _inherit = 'account.payment.line.create'
+
+    @api.multi
+    def _prepare_move_line_domain(self):
+        domain = super(PaymentOrderCreate, self)._prepare_move_line_domain()
```
Comment (pedrobaeza) [inline account_payment_transfer_reconcile_batch/wizard/account_payment_line_create.py]: You can test this creating a new payment order and seeing that the existing invoice is not added again to the new payment order.

Comment (sergio-teruel) [inline account_payment_transfer_reconcile_batch/wizard/account_payment_line_create.py]: @pedrobaeza changes done

**Review by pedrobaeza — APPROVED** [review]
Please squash last 4 commits

**Comment by sergio-teruel** [comment]
@pedrobaeza Squash done.

**Review by rafaelbn — APPROVED** [review]
Thanks!

**Pull request merged**